apps/docs/src/examples/test/Box.svelte

```diff
@@ -1,32 +1,32 @@
 <script lang="ts">
-  import { forwardEventHandlers, T } from '@threlte/core'
+  import { T } from '@threlte/core'
   import { useCursor } from '@threlte/extras'
   import { spring } from 'svelte/motion'
 
-  let color = 'white'
+  let { ...props } = $props()
+
+  let color = $state('white')
 
   const scale = spring(1)
-  const component = forwardEventHandlers()
 
   const { onPointerEnter, onPointerLeave } = useCursor()
 </script>
 
 <T.Group
   scale={$scale}
-  {...$$restProps}
-  bind:this={$component}
+  {...props}
 >
   <T.Mesh
     position.y={0.5}
-    on:pointerenter={onPointerEnter}
-    on:pointerleave={onPointerLeave}
     on:pointerenter={() => {
```

Comment:
oh, what was the situation with `on:___` vs `on___`. Is that something to think about here or not yet?

Reply:
I set up the next branch so that users wouldn't have to migrate or change any events yet (as long as they're not doing things like forwarding events). 

The idea was to defer thinking about how to migrate events to threlte 9. @grischaerbe may want to chime in though

Reply:
In this specific example, multiple duplicate events listeners won't work anymore due to svelte's internals, we could potentially add more signaling to users by giving specific error messages